packaging/rpm/cvmfs-universal.spec

```diff
@@ -83,6 +83,13 @@ BuildRequires: pkgconfig
 BuildRequires: python-devel
 BuildRequires: unzip
 
+%if 0%{?el4} || 0%{?el5} || 0%{?el6}
```

Comment:
We should have this if statement only once: somewhere at the beginning of the file we should decide whether this is a python 2 or a python 3 platform and set a custom variable.